static/js/controllers/edit-user.js

```diff
@@ -125,7 +125,7 @@
       };
 
       var getRoles = function(type, includeAdmin) {
-        if (includeAdmin && type === 'admin') {
+        if (includeAdmin && type === '_admin') {
```

Comment:
Ahh! I answer this myself! This is added with a migration script: https://github.com/medic/medic-api/blob/master/migrations/extract-user-settings-roles.js

Sounds like my code is on the right track.